WooCommerce/WooCommerceTests/Mockups/MockProduct.swift

```diff
@@ -18,6 +18,7 @@ final class MockProduct {
                  fullDescription: String? = "<p>This is the party room!</p>\n",
                  productShippingClass: ProductShippingClass? = nil,
                  backordersSetting: ProductBackordersSetting = .notAllowed,
+                 externalURL: String? = "http://somewhere.com",
```

Comment:
nit: 
```suggestion
                 externalURL: String? = "https://example.com",
```

Reply:
updated in: https://github.com/woocommerce/woocommerce-ios/pull/2432/commits/d565e71f13a7da50b9ce91d470f9f894c93a70fe